app/styleguides/_default/components/subheader.css

```
@@ -5,4 +5,43 @@ $helvetica-stack: Helvetica, sans-serif;
   color: $black;
   font: bold 22px $helvetica-stack;
   margin: 0 0 1em;
+  .h4 {
```

Please add a new line above this one for code block separation
```suggestion
  
  .h4 {
```